docs/modules/installation.rst

```diff
@@ -4,6 +4,13 @@ Installation
 Django installation process
 ---------------------------
 
+These steps assume you already have [the repo](https://github.com/UMNLibraries/racial_covenants_processor/tree/main) cloned and the following installed:
+- Python 3.12
```

Comment:
@mikejcorey for your reference: all my additional dependency issues turned out to be because I was using 3.13. My b! (I also didn't realize pipenv install won't error if the python version required in the pipfile isn't available)